specification/sql/resource-manager/readme.md

```diff
@@ -127,6 +127,25 @@ input-file:
 override-info:
   title: SqlManagementClient
 ```
+### Tag: schema-2017-03-preview
+
+These settings apply only when `--tag=schema-2017-03-preview` is specified on the command line.
+
+These can be regenerated by running the following PowerShell script from this readme file's folder: `dir .\Microsoft.Sql\2017-03-01-preview\ -File | Resolve-Path -Relative | % { " - $_".Replace("\", "/") }`
+
+``` yaml $(tag) == 'schema-2017-03-preview'
+input-file:
+ - ./Microsoft.Sql/2017-03-01-preview/DatabaseCRUD.json
+ - ./Microsoft.Sql/2017-03-01-preview/ExportDatabase.json
+ - ./Microsoft.Sql/2017-03-01-preview/ListDatabasesByElasticPool.json
+ - ./Microsoft.Sql/2017-03-01-preview/PauseDatabase.json
+ - ./Microsoft.Sql/2017-03-01-preview/RenameDatabase.json
+ - ./Microsoft.Sql/2017-03-01-preview/ResumeDatabase.json

```

Comment:
I think you need to regenerate this list

Reply:
Fixed.